cmd/config.go

```diff
@@ -28,6 +28,7 @@ type config struct {
 	Network      networkConfig
 	Log          string
 	SponsorToken string
+	Community    bool
```

Comment:
Ich finde den Namen der Option noch zu generisch. `community` ist ja ein breiter Begriff. Ich hätte erstmal nicht erwartet, dass das dazu führt, dass meine Instanz Daten sendet. Im Titel hier steht "Community Virtual Power Plant". Das ist für nen Confignamen zu lang, aber vielleicht finden wir ja was dazwischen.

Ich finde der Name sollte auch ausdrücken, dass wir hier nur anonyme Daten **sammeln** und **nicht in die Steuerung eingreifen**. Tesla, Elli, Sonnen und co. nennen ihre netzdienlichen Ladelösungen ja auch "Virtual Power Plants". Das ist ja nicht das was wir hier vor haben.

Ideen: `shareChargingData`, `publishStats`, `communityMetrics`, `contributeChargingMetrics`, `shareMetrics` oder irgend eine Kombination aus diesen Wörtern :D